service/src/test/java/uk/gov/hmcts/reform/fpl/service/orders/ManageOrdersEventBuilderTest.java

```diff
@@ -24,30 +36,51 @@
     private final List<Element<GeneratedOrder>> orders = wrapElements(order);
 
     private final SealedOrderHistoryService historyService = mock(SealedOrderHistoryService.class);
-    private final ManageOrdersEventBuilder underTest = new ManageOrdersEventBuilder(historyService);
+    private final AmendedGeneratedOrderFinder amendedGeneratedOrderFinder = mock(AmendedGeneratedOrderFinder.class);
+    private final List<AmendedOrderFinder<? extends AmendableOrder>> finders = List.of(
```

Comment:
this can just be
```java
    private final AmendedOrderFinder<?> finder = mock(AmendedOrderFinder.class);
    private final ManageOrdersEventBuilder underTest = new ManageOrdersEventBuilder(historyService, List.of(finder));
```
and then you just need to mock the interaction with that one finder